.github/workflows/pr-test.yaml

```diff
@@ -90,7 +94,7 @@ jobs:
               },{
                 exist: true,
                 title: /Test format-error/g,
-                state: /`failed`[^`]+`success`/g
+                state: /`success`[^`]+`success`/g
```

Comment:
Why did this test change? My understanding is we are looking for the format check to fail.

This should have a `failed` for the Format check.
![image](https://github.com/user-attachments/assets/bb8ff680-50b3-429e-a5b7-302a2200f153)

Reply:
You are completely right - it was completey my misunderstanding. I did not realize that the format formatted this and therefore broke the test. I will revert my changes.